Description: Extend process dependency to allow for GHC 7.4.1
Author: Joachim Breitner <nomeata@debian.org>

---
The information above should follow the Patch Tagging Guidelines, please
checkout http://dep.debian.net/deps/dep3/ to learn about the format. Here
are templates for supplementary fields that you might want to add:

Origin: <vendor|upstream|other>, <url of original patch>
Bug: <url in upstream bugtracker>
Bug-Debian: http://bugs.debian.org/<bugnumber>
Bug-Ubuntu: https://launchpad.net/bugs/<bugnumber>
Forwarded: <no|not-needed|url proving that it has been forwarded>
Reviewed-By: <name and email of someone who approved the patch>
Last-Update: <YYYY-MM-DD>

--- haskell-haskore-0.2.0.2.orig/haskore.cabal
+++ haskell-haskore-0.2.0.2/haskore.cabal
@@ -1,5 +1,5 @@
 Name:           haskore
-Version:        0.2.0.2
+Version:        0.2.0.2.1
 License:        GPL
 License-File:   LICENSE
 Author:         Paul Hudak <paul.hudak@yale.edu>, Henning Thielemann
@@ -67,7 +67,7 @@ Library
       array >=0.1 && <1.0,
       containers >=0.1 && <1.0,
       random >=1.0 && <2.0,
-      process >=1.0 && <1.1
+      process >=1.0 && <1.2
   Else
     Build-Depends:
       base >= 1.0 && < 2,
